src/serde/ser/error.rs

```diff
@@ -1,7 +1,5 @@
-use std::{
-    fmt::{Display, Formatter},
-    io,
-};
+use serde::__private::Formatter;
```

Comment:
wrong import

Reply:
__private::Formatter is needed, this was an issue from some time ago but this import randomly broke and suddenly needed the __private module before it for some reason, I can test if it is still needed.